src/sentry/south_migrations/0456_auto__add_field_savedsearch_is_global__chg_field_savedsearch_project__.py

```diff
@@ -0,0 +1,1286 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+from sentry.utils.db import is_postgres
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = True
+
+    def forwards(self, orm):
+
+        # Changing field 'SavedSearch.project'
+        # Adding field 'SavedSearch.is_global'
+        if is_postgres():
+            db.execute("ALTER TABLE sentry_savedsearch ALTER COLUMN project_id DROP NOT NULL")
+            db.execute("ALTER TABLE sentry_savedsearch ADD COLUMN is_global boolean NULL")
+            # Set the default separately so that we don't apply to existing rows
+            db.execute("ALTER TABLE sentry_savedsearch ALTER COLUMN is_global SET DEFAULT False;")
```

Comment:
guessing it doesn't matter, but you have a semi colon here

Reply:
also, looks like this was something matt was potentially concerned about? i don't know whether it will be problematic or not, so if you haven't already, maybe check in with him on this

Reply:
"this" meaning the setting the default at the db level not the semi colon 😄